test/SlowTests/Server/Documents/ETL/EtlTimeSeriesTests.cs

```diff
@@ -2004,7 +2004,7 @@ function loadTimeSeriesOfUsersBehavior(docId, counter)
                 var actualUsers = await session.Query<User>().ToArrayAsync();
                 foreach (var user in actualUsers)
                 {
-                    var ts = session.TimeSeriesFor(user.Id, timeSeriesName).GetAsync(DateTime.MinValue, DateTime.MaxValue);
+                    var ts = await session.TimeSeriesFor(user.Id, timeSeriesName).GetAsync(DateTime.MinValue, DateTime.MaxValue);
```

Comment:
let's protect the `GetAsync` with the `AsyncTaskHolder`?

Reply:
https://issues.hibernatingrhinos.com/issue/RavenDB-22590/Introduce-AsyncTaskHolder-to-TimeSeriesSessionOperations